packages/wallet/src/KeyManagement/util/deviceConnection.ts

```diff
@@ -0,0 +1,51 @@
+import { CommunicationType, DeviceType } from '../types';
```

Comment:
I think it would be better to convert LedgerKeyAgent to a 'directory module' and put related utils under that directory

Reply:
plus a `types.ts` to keep the existing file focused on the abstract base